scheduler/scheduler-server/src/test/java/functionaltests/db/schedulerdb/TestJobRemove.java

```diff
@@ -179,6 +182,29 @@ public void testFullDataRemove1() throws Exception {
         checkAllEntitiesDeleted();
     }
 
+    @Test
+    public void testMultipleDataRemoved() throws Exception {
+        TaskFlowJob jobDef = new TaskFlowJob();
+        JavaTask task1 = new JavaTask();
+        task1.setName("task1");
+        task1.setExecutableClassName(TestDummyExecutable.class.getName());
+        jobDef.addTask(task1);
```

Comment:
Regarding the few lines bellow, is the task launched 5 times ?

Reply:
Yes, it is submitting the same job 5 times